src/index.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import * as vscode from 'vscode';
+import { captureStartupPerformance } from './captureStartupPerformance';
+
+const registerCommands = (
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  extensionContext: vscode.ExtensionContext
+): vscode.Disposable => {
+  const captureStartupPerformanceCmd = vscode.commands.registerCommand(
+    'sf.capture.startup.performance',
+    captureStartupPerformance
+  );
+
+  return vscode.Disposable.from(captureStartupPerformanceCmd);
```

Comment:
register command returns a disposable already and vscode.Disposable.from is used to "Combine many disposable-likes into one. You can use this method when having objects with a dispose function which aren't instances of Disposable."
```suggestion
  return vscode.commands.registerCommand(
    'sf.capture.startup.performance',
    captureStartupPerformance
  );
```